backend/app/api/content_machine.py

```
@@ -0,0 +1,314 @@
+            commentary_style=commentary_style,
+            writer_style=request.writer_style,
+        )
+        
+        return GenerateWithQuoteResponse(
+            content=result["content"],
+            quote_used=result["quote"],
+            source=result["source"],
+            commentary_style=result["commentary_style"],
+            character_count=result["character_count"],
+        )
+    except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)